### ISSUE TITLE
initial state of reducers can not be null

### ISSUE THREAD

**Issue by littleStudent** [btroncone/ngrx-store-logger#1]
I have a reducer which is initialised with `null`. This leads to the following error:

`ORIGINAL EXCEPTION: TypeError: Cannot read property 'started' of null`

This is the line that is crashing: [Line](https://github.com/btroncone/ngrx-store-logger/blob/master/src/index.ts#L117). 

I think the logger should be able to handle null values. Maybe its enough to wrap it in a if-clause because initialising with null does not have to get logged anyways?

**Comment by btroncone** [comment]
I am looking into this, can you share an example that is failing? I am having trouble reproducing this with a reducer being initialized with `null`. Thanks in advance!

**Comment by littleStudent** [comment]
I did some more debugging.
The problem occurs if you use `ngrx-store-logger` and `ngrx/devtools`
ngrx-store-logger is calling `postLogger`, without calling `preLogger` for the first log statment.
afterwards it seems to work.

my setup looks like this

``` Javascript
return browser.bootstrap(App, [
        ...
        provideStore({ location }),
        loggerMiddleware(loggerOptions),
        instrumentStore(),
        ...
    ])
```

if i remove `instrumentStore()` everything works fine.

**Comment by btroncone** [comment]
Thanks, that may have taken me a bit to track down. Let me run through some testing and I will publish a fix tonight. Thanks again!

**Comment by btroncone** [comment]
This has been fixed as of v0.0.8, thanks!
